code/modules/mob/living/simple_animal/hostile/headcrab.dm

```diff
@@ -47,9 +47,13 @@
 			if(C.status_flags & XENO_HOST)
 				src << "<span class='userdanger'>A foreign presence repels us from this body. Perhaps we should try to infest another?</span>"
 				return
+			if(getorgan(/obj/item/organ/internal/body_egg/changeling_egg))
+				src << "<span class='userdanger'>One of us has already planted thier seed in this body! Perhaps we should try to infest another?</span>"
+				return
 			Infect(target)
 			src << "<span class='userdanger'>With our egg laid, our death approaches rapidly...</span>"
 			spawn(100)
+				egg_lain = 0
```

Comment:
The idea is, normally if a headslug plants an egg in something, then someone gibs the body, they can't return. This allows a fellow ling to potentially revive the headslug with a lazarus injector and have them plant the egg again. If it didn't set the egg_lain to 0 it'd be unable to plant a new one on respawn. Get it?